charts/schema-registry/templates/_helpers.tpl

```diff
@@ -0,0 +1,37 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "schema-registry.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" }}
+{{- end }}
+
+{{- define "schema-registry.configmap" -}}
+{{- $name := default .Chart.Name .Values.nameOverride -}}
+{{- printf "%s-%s-configmap" .Release.Name $name | trunc 63 | trimSuffix "-" -}}
+{{- end }}
```

Comment:
This seems redundant, why not just set the name to:
`name: {{ include "schema-registry.name" . }}-configmap` ?